backend/tarsy/integrations/llm/client.py

```
@@ -399,6 +411,17 @@ async def _execute_with_retry(
                 
                 return response, usage_metadata
                 
+            except asyncio.TimeoutError:
+                # Handle timeout specifically - don't retry as it's likely a systemic issue
+                logger.error(f"LLM API call timed out after {timeout_seconds}s (attempt {attempt + 1}/{max_retries + 1})")
+                if attempt < max_retries:
+                    logger.warning(f"Retrying after timeout in 5s...")
```

Ruff (0.14.0)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)